armory/metrics/task.py

```diff
@@ -818,6 +828,107 @@ def object_detection_mAP(y_list, y_pred_list, iou_threshold=0.5, class_list=None
     return np.fromiter(ap_per_class.values(), dtype=float).mean()
 
 
+def armory_to_tide_ground_truth(y_dict, image_id, data_ground_truth):
+    for y in [dict(zip(y_dict, t)) for t in zip(*y_dict.values())]:
+        x1, y1, x2, y2 = y["boxes"]
+        width = abs(x1 - x2)
+        height = abs(y1 - y2)
+        x_min = min(x1, x2)
+        y_min = min(y1, y2)
+
+        y_tidecv = {
+            "image_id": image_id,
+            "class_id": y["labels"],
+            "box": [x_min, y_min, width, height],
+        }
+        data_ground_truth.add_ground_truth(**y_tidecv)
+
+
+def armory_to_tide_detection(y_dict, image_id, data_detection):
+    for y in [dict(zip(y_dict, t)) for t in zip(*y_dict.values())]:
+        x1, y1, x2, y2 = y["boxes"]
+        width = abs(x1 - x2)
+        height = abs(y1 - y2)
+        x_min = min(x1, x2)
+        y_min = min(y1, y2)
+
+        y_tidecv = {
+            "image_id": image_id,
+            "class_id": y["labels"],
+            "box": [x_min, y_min, width, height],
+            "score": y["scores"],
+        }
+        data_detection.add_detection(**y_tidecv)
```

Comment:
This is almost a complete duplicate of the previous function. Can you combine, and maybe add a bool for `is_detection`, and use that to determine whether to add the `"score": y["scores"],` line?

Reply:
agree - will update

Reply:
resolved with [851cfcc](https://github.com/twosixlabs/armory/pull/1762/commits/851cfcc51a72d714b1781d9dcf0de4ac81c52234)